docs/spark/spark-writes.md

```diff
@@ -311,7 +311,11 @@ distribution & sort order to Spark.
 {{< /hint >}}
 
 {{< hint info >}}
-Both global sort (`orderBy`/`sort`) and local sort (`sortWithinPartitions`) work for the requirement.
+Both global sort and local sort (data sorted within each partition) can be used to write against partitioned table.
```

Comment:
Do we have global or local defined anywhere? I still think we should have something like
"global sort (sorting all the data in the write) and local sort (sorting the data within a Spark task)"

Reply:
I've simply changed the description, perhaps we can go into more detail in this separate doc:
https://github.com/apache/iceberg/pull/4463#discussion_r856268809